conda/recipes/libcudf/meta.yaml

```diff
@@ -45,7 +45,7 @@ requirements:
     - cuda-nvrtc ={{ cuda_version }}
     - cuda-nvrtc-dev ={{ cuda_version }}
     - cuda-nvtx ={{ cuda_version }}
-    - libcurand-dev
+    - libcurand-dev {{libcurand_version}}
```

Comment:
We must pin both libcurand and libcurand-dev. Follow raft’s pinnings and naming scheme: https://github.com/rapidsai/raft/blob/a98295b516ef58bc855177077860bab2a2a76d77/conda/recipes/libraft/meta.yaml#L58
```suggestion
    - libcurand-dev {{ libcurand_version }}
```

Reply:
We're discussing this offline, but for context I had done the minimal fix here to unblock release while investigating exactly where we need libcurand. I'm fairly certain it's only needed in benchmarks.

Reply:
To be complete I went ahead and added the run pinnings for libcudf-tests. I'm guessing that nobody actually runs the benchmarks from the package, which is why no issues have ever been observed with the missing requirements.